clientv3/concurrency/example_election_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2017 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package concurrency_test
```

Comment:
Why this file is not a symlink to the file in tests module?

Reply:
Good catch: Fixed: https://github.com/etcd-io/etcd/pull/12339/commits/dd45d04b2d7cf07c27f6322dfb794efae84978ea